src/components/PostDetails.tsx

```diff
@@ -1,118 +1,119 @@
 import React from 'react';
+import { Post } from '../types/Post';
+import { Comment } from '../types/Comment';
 import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
+import { deleteComment } from '../api/api';
 
-export const PostDetails: React.FC = () => {
-  return (
-    <div className="content" data-cy="PostDetails">
-      <div className="content" data-cy="PostDetails">
-        <div className="block">
-          <h2 data-cy="PostTitle">
-            #18: voluptate et itaque vero tempora molestiae
-          </h2>
-
-          <p data-cy="PostBody">
-            eveniet quo quis
-            laborum totam consequatur non dolor
-            ut et est repudiandae
-            est voluptatem vel debitis et magnam
-          </p>
-        </div>
+type Props = {
+  selectedPost: Post;
+  isLoadingComments: boolean;
+  error: boolean;
+  setError: (v: boolean) => void;
+  comments: Comment[];
+  setComments: React.Dispatch<React.SetStateAction<Comment[]>>;
+  activeForm: Post | null;
+  setActiveForm: (v: Post | null) => void;
+};
 
-        <div className="block">
-          <Loader />
+export const PostDetails: React.FC<Props> = ({
+  isLoadingComments, error, setError, selectedPost, comments, setComments,
+  activeForm, setActiveForm,
+}) => {
+  const handleDelete = (commentId: number) => {
+    setComments(currentComments => currentComments.filter(
+      currentComment => currentComment.id !== commentId,
+    ));
 
-          <div className="notification is-danger" data-cy="CommentsError">
-            Something went wrong
-          </div>
+    deleteComment(commentId)
+      .then()
+      .catch(() => setError(true))
+      .finally();
```

Comment:
You have a .finally() block with no functionality inside after calling 'deleteComment' function. This is redundant and should be removed unless a specific finalization operation is to be carried out regardless of promise fulfillment/rejection.